README.md

```diff
@@ -33,7 +33,20 @@ It’s a React.js and Typescript-based browser application with a a Node.js back
     ```sh
     yarn run start
     ```
+    
+## Running tests
 
+To run all tests once
+
+    ```sh
+    yarn test
+    ```
+    
+while to start the test runner in watch mode
+
+    ```sh
+    yarn dev
+    ```
```

Comment:
These shouldn’t be indented—GH is rendering them poorly as a result:

![screen shot 2017-06-08 at 6 19 55 pm](https://user-images.githubusercontent.com/74178/26957064-42dce396-4c77-11e7-864c-327197084b9b.png)

(FWIW: code blocks are generally indicated by *either* a 4-space indent or triple backticks, so combining them makes the parser think the backticks are part of the code, not *demarcating* the code. The exception is when they’re in a list, where a 4-space indent instead means the item is part of the same list item even though it’s a separate paragraph—if you indented by 8-spaces you’d see the same problem again.)